HHS/OPRE-OPS
# portfolio feedback

## What changed

- adds hover styles for table headings
- adds hover styles for Hero `read more`
- refactors `read more` to a `<button/>`

## Issue

Feedback from today's SU

## How to test

Go to Portfolio page > Projects and Spending Tab > and hover over table headings.
Also see Hero section `read more` is now a button and more a11y friendly and has a hover style.

## Screenshots

<img width="1158" alt="SCR-20230307-lpx" src="https://user-images.githubusercontent.com/4629398/223559945-8bbe4357-1325-46fb-9611-a6476b685031.png">
